Common/Python/VolatilityModelPythonWrapper.cs

```diff
@@ -58,7 +58,7 @@ public override decimal Volatility
             {
                 using (Py.GIL())
                 {
-                    return (decimal)_model.Volatility;
+                    return (decimal)(double)_model.Volatility;
```

Comment:
Using `NumberStyles.Number | NumberStyles.AllowExponent` in the PythonNet converter seems to solve the issue too https://github.com/QuantConnect/pythonnet/blob/master/src/runtime/converter.cs#L944 . The issue triggers when the `string` representation uses the exponent format. I think it might be better to change the converter directly so it convers any other use case too